lib/src/model/tile_model.dart

```diff
@@ -64,10 +65,16 @@ class TileFeature {
           _paths = decodePolygons(geometry).toList(growable: false);
           break;
       }
-      _geometry = null;
```

Comment:
Leaving geometry in-place comes with a significant memory penalty. Why do we need to do this?

Reply:
Hey, if we keep the geometry in memory and the user loads the tile again (eg. moves back) the cached geometry is used instead of decoding it again. Or am I missing something?

Reply:
Hey, wups jep, overlooked it. It makes sense to delete the geometry after decoding.